net-analyzer/gsad/gsad-21.4.4-r1.ebuild

```diff
@@ -0,0 +1,106 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake systemd toolchain-funcs
+
+DESCRIPTION="Greenbone Security Assistant"
+HOMEPAGE="https://www.greenbone.net https://github.com/greenbone/gsad"
+SRC_URI="https://github.com/greenbone/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+SLOT="0"
+LICENSE="AGPL-3+"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc"
+
+DEPEND="
+	acct-user/gvm
+	dev-libs/glib:2
+	dev-libs/libgcrypt:0=
+	dev-libs/libxml2
+	dev-libs/libxslt
+	>=net-analyzer/gvm-libs-${PV}
+	net-libs/gnutls:=
+	net-libs/libmicrohttpd:=
+"
+RDEPEND="
+	${DEPEND}
+	>=net-analyzer/gvmd-${PV}
+	>=net-analyzer/gsa-${PV}
+	net-analyzer/ospd-openvas
+"
+BDEPEND="
+	dev-python/polib
+	virtual/pkgconfig
+	doc? (
+		app-doc/doxygen[dot]
+		app-doc/xmltoman
+		app-text/htmldoc
+		sys-devel/gettext
+	)
+"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-21.4.4-r1-fix-PCRE2-user-validation-regex.patch
```

Comment:
What is the source of those patches? Are the already upstreamed? Or is there maybe a pending PR for them?

Reply:
I have backported this https://github.com/greenbone/gsad/commit/62c4c9180e8f94e0b148dbba15cca9d886079f57
and posted the patch in a br: https://bugs.gentoo.org/905411
I think the patch is necessary if we want to keep version 21.4 in the tree.

I really think we could remove 21.4 because it is obsolete and marked as obsolete in the same GVM scan results, but I'm still waiting for a reply from @j-licht to add me as a proxied maintainer.

Reply:
I think you can go ahead and add yourself.

Reply:
@thesamesam I added mysql as co-maintainer and I squashed some commits (I hope I didn't make any mistake doing this)